### PR TITLE
Restapi 567 fix some parameter validations and other minor bugs

### PR DESCRIPTION
**Pull request by jpdorsch**
This branch fixes:

- In `cscs_api_common.py` file, the reference to `DEBUG_MODE` environment variable must be changed to `F7T_DEBUG_MODE`.
- `list_jobs method` in `compute.py` verifies that `jobs` param only contains numeric values.
- `post_upload_request` in `storage.py` verifies that `sourcePath` param is not null or empty and that the file exists.
- `internal_cp/mv/rm/rsync` methods in `storage.py` verifies that `sourcePath` param and `targetPath` are not null or empty.

Applies new feature
- Implement authorization/authentication "decorator" `check_auth_header` to avoid duplicated code.  This decorator is replicated in `certificator.py` (sysadmin deployment suggestion).

Changes:
- `F7T_STORAGE_MAX_FILE_SIZE` environment variable is expressed in MBytes to match with `F7T_UTILITIES_MAX_FILE_SIZE`.

Deprecated:
- Remove `PUT storage/xfer-internal/upload` method handler

### CODE REVIEW

File: src/status/status.py [hunk 1]
```diff
@@ -299,19 +273,16 @@ def status_service(servicename):
     return jsonify(service=servicename,status=status,description=description), 200
 
 
+from functools import wraps
```
Comment (ekouts) [inline src/status/status.py:276]: Do you need this import here?

**Comment by ekouts** [comment]
@jpdorsch Since you are removing the `PUT storage/xfer-internal/upload` method handler, we should remove it from openapi as well. I think it's better in this PR, since it is going to `dev` but I can also change it in the PR I have open about the /users directory in /storage (that will be merged directly to `master`)?

File: src/status/status.py (hunk 1 above)
Comment (jpdorsch) [reply src/status/status.py:276]: Forgot to remove it.

**Comment by jpdorsch** [comment]
It's better to do it on my PR, since it belongs to this topics. And it's already deprecated on OpenAPI server.

**Comment by jpdorsch** [comment]
@fcndl thanks for your feedback, all fixed.
There were mostly some missing checks on paths and in jobs (when passed as a list).

About `rm` the problem was that we use the command `rm -rf <dir>`. Option `-f` forces to delete without prompt (so we avoid the use of `echo yes`), but doesn't inform of a possible error (even on non existing directories). I've replaced it by `--interactive=never` that doesn't prompt neither, but let you know of an error.

If everything's alright we can merge.

**Pull request merged**